docs/apache-airflow/howto/set-up-database.rst

```diff
@@ -300,6 +300,14 @@ We recommend using the ``mysqlclient`` driver and specifying it in your SqlAlche
 
 We also support the ``mysql-connector-python`` driver, which lets you connect through SSL
 without any cert options provided.
+Unfortunately, this package depends on outdated version of ``protobuf<4.0``  which is no longer supported.
```

Comment:
I think we should just explain that we have an extra option for another mysql client and in the extra info below we can explain that users should be aware of proto issue with link to the proto update request we submitted to the lib.

Reply:
Good point, fixed.